docs(design): Add general content guidelines for error messages (#1958)
GetJobber/atlantis: docs/content/voice-and-tone.stories.mdx

| ----------------------------- | ----------------------------------------- |
| No search results for "Brian" | Could not detect match for string "Brian" |

### Error messages should be helpful

#### Explain

Explain what happened, and how to address the issue in a concise manner. If you
can't be specific, be general without being overly technical.

Avoid blaming the user in form validations - focus on the fix.

Empathize with the user - this probably isn't a great time for jokes.

| ✅ **Do**                                                          | ❌ **Don't**                                  |
| ------------------------------------------------------------------ | --------------------------------------------- |

> | Client already has the phone number 555-5555                       | Duplicate value detected                      |

Author: ```suggestion
| 555-555-5555 is already on file for this client                      | Duplicate value detected                      |
```
